paths/keys/destroy-collection.yaml

```diff
@@ -24,6 +24,9 @@ parameters:
       <li><code>updated_at:{>=|<=}2013-02-21T00:00:00Z</code> for date range queries</li>
       <li><code>unmentioned_in_upload:upload_id</code> to filter keys unmentioned within upload</li>
     </ul>
+    <br/>
+    <p><strong>Caution:</strong> If an empty value is sent for any of these filters, all records will be deleted as it is treated as selecting all records. Please ensure a specific value is provided to avoid accidental data loss.</p>
```

Comment:
Or how about? Otherwise it might sound you have to provide a value for every single filter 😅 
> Parameters with empty values will be ignored and have the same effect as if the parameter is not present

Reply:
True.